actions/scoring.js

```diff
@@ -92,7 +104,7 @@ async function getCompoundInteractions(address, chainId) {
     return response.data.data.items.length;
   } catch (e) {
     console.log("Error fetching Compound data: " + e);
-    toast.error("Error fetching Compound data. See console.");
+    // toast.error("Error fetching Compound data. See console.");
```

Comment:
`console.error` could be called instead in this instance.
Toast call must be uncommented. And placed after error logs with a user-friendly message.